### PR TITLE
📖 Update README.md

### PR DESCRIPTION
**Pull request by olivekl**
Add risk levels to table of Scorecards Checks section; remove lists of each risk level in the Scoring section. (To streamline navigation; keeps the same info but just in shorter format)

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update, no breaking changes

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,26 +348,26 @@ indicates the check was completely sure of the result.
...
-[Security-Policy](docs/checks.md#security-policy)             | Does the project contain a [security policy](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository)?
-[Signed-Releases](docs/checks.md#signed-releases)             | Does the project cryptographically [sign releases](https://wiki.debian.org/Creating%20signed%20GitHub%20releases)?
-[Token-Permissions](docs/checks.md#token-permissions)           | Does the project declare GitHub workflow tokens as [read only](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)?
-[Vulnerabilities](docs/checks.md#vulnerabilities)             | Does the project have unfixed vulnerabilities? Uses the [OSV service](https://osv.dev).
+Name                        | Description | Risk Level
```
Comment (naveensrinivasan) [inline README.md:351]: IMO this table could be sorted by `High`, `Medium` and `Low`

**Review by naveensrinivasan — APPROVED** [review]
I will let others chime in. Thanks!

File: README.md (hunk 1 above)
Comment (laurentsimon) [reply README.md:351]: Good idea, and then alphabetically within each risk level.

**Review by laurentsimon — APPROVED** [review]
Thanks @olivekl

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

**Pull request merged**